clustering/singleton/WFLY-11098_Singleton_Service_Election_Listener.adoc

```diff
@@ -0,0 +1,60 @@
+= Singleton Service Election Listener
+:author:            Paul Ferraro
+:email:             paul.ferraro@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+The MSC includes a listener facility to inform a listener when a given service starts and stops.
+For singleton services, we need similar facility - to inform a listener, not just when its service starts and stops, but also which cluster member was elected as the new primary provider.
+Users will register this listener when the service is installed.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-11098
+
+=== Related Issues
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+* mailto:tborgato@redhat.com[Tommaso Borgato]
+
+=== Affected Projects or Components
+
+Clustering
+
+=== Other Interested Projects
```

Comment:
Quickstarts should be added as interested project as there's already a ha-singleton-service QS which can be expanded to show the listeners per https://issues.jboss.org/browse/WFLY-11836